doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -168,16 +167,18 @@ nRF5340 Audio
 nRF Desktop
 -----------
 
-* Added an application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
-* Added application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
-  See the debug configuration section of the :ref:`nrf_desktop` application for more details.
+* Added:
 
-* Changed:
+  * An application log informing about the configuration option value update in the :ref:`nrf_desktop_motion`.
+  * Application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL<config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
```

Comment:
```suggestion
  * Application-specific Kconfig options (:ref:`CONFIG_DESKTOP_LOG <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_SHELL <config_desktop_app_options>`) to simplify the debug configurations for the Logging and Shell subsystems.
```
Do we need to have the Kconfig options in the brackets? I would leave the brackets out.